atoma-state/src/types.rs

```diff
@@ -115,6 +115,13 @@ pub struct CheapestNode {
     pub max_num_compute_units: i64,
 }
 
+/// Response for getting the node distribution
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize, FromRow)]
```

Comment:
Maybe expand on the docs here

Reply:
check the new comment, and let me know what you think

Reply:
Much better honestly